.devops/deploy-pipelines.yml

```
@@ -555,17 +812,71 @@ stages:
               notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
               instructions: "Please approve or reject PROD promotions for release $(release_url)"
               onTimeout: 'skip'
-  - stage: "Prod_RollbackToLatestRelease"
-    displayName: 'PROD rollback to the latest Release'
-    dependsOn: [ Prod_RollbackForApproval ]
-    condition: succeeded()
+  # STEP 0: scale up staging release (by removing annotation on staging release scaledObject)
+  - stage: Scale_Up_Staging_Deployment
+    displayName: Scale up STAGING deployment
+    pool:
+      name: pagopa-prod-linux
     jobs:
-      - job: Prod_Rollback
-        pool:
-          name: pagopa-prod-linux
+      - job: Scale_Up_Staging_Deployment
+        displayName: Remove annotation autoscaling.keda.sh/paused-replicas="0" from staging release scaledObject
+        steps:
+          - task: Kubernetes@1
+            displayName: Remove annotation autoscaling.keda.sh/paused-replicas="0" from staging release scaledObject
+            continueOnError: true
+            inputs:
+              kubernetesServiceEndpoint: $(PROD_KUBERNETES_SERVICE_CONN)
+              kubernetesCluster: $(kubernetesCluster)
+              namespace: ecommerce
+              command: scale
+              arguments: --replicas=1 deployment/pagopaecommercetxschedulerservice-microservice-chart
```

watch out! the deployment to be scaled up is the staging one
moreover the `pagopaecommercetxschedulerservice` actually can be retrieved by the `$(K8S_IMAGE_REPOSITORY_NAME)` variable interpolation, so the command will become 
```suggestion
              arguments: --replicas=1 deployment/$(K8S_IMAGE_REPOSITORY_NAME)-staging-microservice-chart
```